tslint.json

```diff
@@ -1,73 +1,64 @@
+        "only-arrow-functions": [true, "allow-declarations", "allow-named-functions"],
 
-    // TODO: Enable these
-    "completed-docs": false,
-    "no-any": false,
-    "no-magic-numbers": false,
-    "no-non-null-assertion": false,
-    "no-null-keyword": false,
-    "no-require-imports": false,
-    "no-unbound-method": false,
-    "no-unnecessary-qualifier": false,
-    "no-use-before-declare": false,
-    "no-void-expression": false,
-    "prefer-function-over-method": false,
-    "strict-type-predicates": false,
-    "triple-equals": [true, "allow-undefined-check"],
-    "typedef": false,
+        // TODO: Enable these
+        "completed-docs": false,
+        "no-any": false,
+        "no-magic-numbers": false,
+        "no-non-null-assertion": false,
+        "no-null-keyword": false,
+        "no-require-imports": false,
+        "no-unbound-method": false,
+        "no-unnecessary-qualifier": false,
+        "no-use-before-declare": false,
+        "no-void-expression": false,
+        "prefer-function-over-method": false,
+        "strict-type-predicates": false,
+        "triple-equals": [true, "allow-undefined-check"],
+        "typedef": false,
 
-    "ban": [true,
-      ["describe", "only"],
-      ["it", "only"]
-    ],
+        "ban": [true, ["describe", "only"], ["it", "only"]],
 
-    "interface-name": false,
-    "file-header": [true,
-      "Copyright \\d{4} Palantir Technologies, Inc."
-    ],
-    "max-classes-per-file": false,
-    "max-line-length": [true, 140],
-    "member-ordering": [true,
-      "public-before-private",
-      "static-before-instance",
-      "variables-before-functions"
-    ],
-    "newline-per-chained-call": false,
-    "no-console": {
-      "options": ["log"]
-    },
-    "no-switch-case-fall-through": true,
-    "strict-boolean-expressions": [true, "allow-boolean-or-undefined"],
-    "switch-default": false,
-    "variable-name": [true,
-      "ban-keywords",
-      "check-format",
-      "allow-leading-underscore",
-      "allow-pascal-case"
-    ],
-    "linebreak-style": false
-  }
+        "interface-name": false,
+        "file-header": [true, "Copyright \\d{4} Palantir Technologies, Inc."],
+        "max-classes-per-file": false,
+        "max-function-body-length": false,
+        "max-line-length": [true, 140],
+        "member-ordering": [
+            true,
+            "public-before-private",
+            "static-before-instance",
+            "variables-before-functions"
+        ],
+        "newline-per-chained-call": false,
+        "no-console": {
+            "options": ["log"]
+        },
+        "no-switch-case-fall-through": true,
+        "strict-boolean-expressions": [true, "allow-boolean-or-undefined"],
+        "switch-default": false,
+        "variable-name": [
+            true,
+            "ban-keywords",
+            "check-format",
+            "allow-leading-underscore",
+            "allow-pascal-case"
+        ],
+        "linebreak-style": false
+    }
```

Comment:
Please try not to include non feature related changes in a PR since it is the top reason for a review to take longer than it needs to. Examples are: prettier, making stuff clearer, fixing typos, etc..

Reply:
@goktugyil thanks for stepping in to review these 😄 I appreciate the effort! I'm waiting on this for a review from another maintainer. This PR was sent back before Prettier was finished being integrated into the repo, so there are likely a bunch of unintended merge conflicts.

We should also note that this repo is being deprecated soon after wavering amounts of maintenance activity (#4534), so it's not likely we'll have time to get through a lot of these older PRs.